.github/workflows/build.yml

```
@@ -98,3 +98,38 @@ jobs:
       - name: Setup tmate debug session
         uses: mxschmitt/action-tmate@v3
         if: env.RUN_TMATE
+  test:
+    name: Run Go Package Tests
```

Use similar test name to [our Python skeleton](https://github.com/cisagov/skeleton-python-library/blob/65efbf63dbfc4f2dfaec9190d1ec9448af565f64/.github/workflows/build.yml#L106):
```suggestion
    name: test source
```